README.md

```diff
@@ -50,6 +50,14 @@ $ gem install cocoapods
 
 To integrate EasyTipView into your Xcode project using CocoaPods, specify it in your `Podfile`:
 
+For **Swift 4.1**
+
+```ruby
+pod 'EasyTipView', '~> 1.1.0'
```

Comment:
2 cents, since this is a breaking change this really should be version `2.0.0`. Anyone still using Swift 3 would be broken by a minor bump. Hopefully this is merged soon!